tests/autoyast/wicked.pm

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2015-2016 SUSE LLC
+# Copyright (C) 2015-2020 SUSE LLC
```

Comment:
s/©/(C)/

Reply:
I think it is correct copyright and no © in the line.